sklearn/tree/tests/test_export.py

```diff
@@ -448,3 +448,14 @@ def test_plot_tree_gini(pyplot):
                                    "samples = 6\nvalue = [3, 3]")
     assert nodes[1].get_text() == "gini = 0.0\nsamples = 3\nvalue = [3, 0]"
     assert nodes[2].get_text() == "gini = 0.0\nsamples = 3\nvalue = [0, 3]"
+
+
+# FIXME: to be removed in 0.25
+def test_plot_tree_rotate_deprecation():
```

Comment:
The pyplot fixture is needed to skip the test when matplotlib is not available:

```suggestion
def test_plot_tree_rotate_deprecation(pyplot):
```

Reply:
Thanks @ogrisel ... but I don't understand why... :(

Reply:
Sorry, found it in `sklearn/conftest.py`... :)
Thanks again!